### DISCUSSION TITLE
[match] import multiple provisioning profile

### DISCUSSION THREAD

**Discussion by Aure77** [fastlane/fastlane#21220] [Ideas]
I would use `fastlane match import` without Apple Dev Portal connection.
I use this command (assume having Matchfile with git storage settings):
```
fastlane match import \
       --readonly true \
       --type adhoc \
       --skip_certificate_matching true
```
Import ask for _.cer_ path (mandatory), _.p12_ path (mandatory) and provisioning profile (PP) (optional).
But In my repo, I have a lot of PP (for app & extension) and match import only support a single path for PP.. 
So I need to run import multiple times and .cer & .p12 ask are redundant...

Ideas:

- `import` should be able to import only PP (without asking for .cer/.p12)
- paths (.cer / .p12 / PP) should be available as command line argument to avoid interactive mode, ex:
```
fastlane match import \
       --readonly true \
       --type adhoc \
       --pp_path "./MyApp.mobileprovision"
       --cer_path "./distribution.cer"
       --skip_certificate_matching true
```

**Comment by BogdanMatran** [comment]
I have create this Plugin https://github.com/BogdanMatran/fastlane-plugin-match_import_multiple
